avm/res/web/serverfarm/main.bicep

```diff
@@ -16,10 +16,10 @@ param name string
   'I1v2'
   '''
 })
-param skuName string
+param skuName string = 'P1v3'
```

Comment:
Hey @tsc-buddy thanks a lot for this update and also for opening and following up on a PSRule fix.

Is it intentional to set this specific value as default? If so, please also:
- change the parameter type to Optional in the description
- remove the parameter from defaults test, as they should only use required parameters (optional parameters are meant to use their defaults in this case)

Reply:
Yes it was intentional, simply to make a default deployment leverage Availability zones.

I can update the test.

Reply:
All good for me if that is the intention @tsc-buddy Just double checking.
I'd still suggest the 2 points above if a default value is set, i.e. make it Optional and remove from defaults test.